editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2661,6 +2665,27 @@ void VisualShaderEditor::_port_edited(const StringName &p_property, const Varian
 	undo_redo->commit_action();
 }
 
+void VisualShaderEditor::_node_ports_removed(int p_id) {
+	VisualShader::Type shader_type = visual_shader->get_shader_type();
+	Ref<VisualShaderNode> node = visual_shader->get_node(shader_type, p_id);
+	if (!node.is_valid())
+		return;
```

Comment:
```suggestion
	if (!node.is_valid()) {
		return;
	}
```
Code style, we use brackets always for if-statements